id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/validator/IdRequestValidator.java

```diff
@@ -128,6 +139,16 @@ public class IdRequestValidator extends BaseIdRepoValidator implements Validator
 	@Autowired
 	private IdRepoServiceHelper idRepoServiceHelper;
 
+	@Autowired
+	@Qualifier("plainRestTemplate")
+	private RestTemplate restTemplate;
+
+	/** The config server file storage URL. */
+	@Value("${config.server.file.storage.uri}")
+	private String configServerFileStorageURL;
+
+	@Value("${identity.service.verifiedattributes.file}")
```

Comment:
Rename property name as per module convention - mosip.idrepo.verified-attributes.schema-url
Take the file URL in a single property, so that if we host the schema from other server it will be easy to update the config.